### ISSUE TITLE
Confirmation that certificate will be replaced

### ISSUE THREAD

**Issue by davidonus** [shibayan/keyvault-acmebot#565]
Great project and have been using it at my company for managing let's encrypt certs.

One thing I have noticed is that if you amass several certificates, it is open to human error in replacing the certificates. 

Unless you manually check through all your current certs you can replace one if you use the same name.

Is there scope to add a check on the add function to check if that name already exists and ask for confirmation that you are happy for it to be replaced?

**Comment by shibayan** [comment]
Good idea. It is not the desired behavior to have certificates overwritten, so it would be better to be able to issue a warning.
